### PR TITLE
updated A to Q1 & Q7

### PR DESCRIPTION
**Pull request by chanan-sack**
Corrected technical ambiguity, simply false misinformation, spelling mistakes and grammar (hope i didn't leave any:) )
This question is super important and is the first one for a very good reason.

the first two paragraphs were (IMO) completely un-called for, if a question starts with "What was the impetus?" the answer starts with "The impetus was" and not with changing the merit of the question. It is the responsibility of the CFTC to ask questions and it's our willingness to answer them that will help, changing the question and adding un-called for lines does not help.

By technical ambiguity and simply false misinformation i refer to Bitcoin lacking programmability and that being the key difference.
This is factually incorrect, fixed the answer by noting Turing completeness as the key change and providing links for what Turing completeness and conditional branching are.

### CODE REVIEW

**Comment by econoar** [comment]
Hm, I see where you're going but not sure I agree on a full cut of the second part. I'm not seeing how that's misinformation. Vitalik himself had stated that he wanted to build on top of Bitcoin but due to it's limitations and the community not wanting to adapt, he moved onto Ethereum. I think it's very important to get that point across in this question.

**Comment by chanan-sack** [comment]
That point is already clear, besides: the question is asking "what was the
reason to create a new network" and the answer should be technical and
clear. We should refrain from highly theoreticall responses because they
don't answer the question

**Comment by chanan-sack** [comment]
why is this not being added?

**Comment by alexfisher** [comment]
Agree with econoar... I understand where these changes are trying to go with providing more objective answers, but as they are don't think they do it justice.  There are major differences between how Bitcoin and Ethereum work (e.g. UTXO vs Accounts, State, the EVM which includes instructions beyond just conditional logic).  Think it needs work before accepting.

**Comment by chanan-sack** [comment]
i understand and agree that my answer can be improved on as well, but the current one is false and that is the problem here.

And although there are more technical differences - this is the one that makes ethereum inherently different by design and nature.

**Comment by chanan-sack** [comment]
Hi, im not really getting an answer here. the current answer is misleading and incorrect. ive offered to change my current one to better fit the picture but it does not seem like i am getting feedback

File: other/ethhub-cftc-response.md [hunk 1]
```diff
@@ -54,9 +59,12 @@ A user must only wait 1 block confirmation for their transaction to be validated
...
 
-Ethereum 2.0 \(Serenity\) is a substantial upgrade for the Ethereum network being deployed in multiple phases. The main features in this upgrade include a move to a novel Proof of Stake system known as Casper \(for securing the network\), sharding \(for scaling the network\) and eWASM \(a new virtual machine for the network\). These upgrades and new features are developed by a decentralized network of core developers spanning 8 individual Ethereum software client teams, the non-profit Ethereum Foundation, and additional volunteer developers and researchers from around the globe.
+In the future additional changes will come to ethereum drawing the technological difference even further:
+ * Serenity is a substantial upgrade for Ethereum being deployed in multiple phases, the main features in this upgrade        include a move to a novel Proof of Stake system known as Casper.
```
Comment (ChazSchmidt) [inline other/ethhub-cftc-response.md]: Casper probably needs more of an explanation here. I'm not sure if it is covered later but they probably want to know a little about slashing.

File: other/ethhub-cftc-response.md [hunk 2]
```diff
@@ -54,9 +59,12 @@ A user must only wait 1 block confirmation for their transaction to be validated
...
+The Ethereum network is a turing-complete blockchain. This means that it is capable of executing arbitrary code - known as smart contracts. The ability to run code on the Ethereum blockchain means that developers can build applications on top of it. Applications are all interoperable within the confines of the Ethereum network. Ethereum acts as a common protocol for all decentralized applications built atop, in the way that TCP/IP acts as a common protocol for all public web-based applications. It is not strictly limited to being used as a settlement layer as Bitcoin has become.
 
-Ethereum 2.0 \(Serenity\) is a substantial upgrade for the Ethereum network being deployed in multiple phases. The main features in this upgrade include a move to a novel Proof of Stake system known as Casper \(for securing the network\), sharding \(for scaling the network\) and eWASM \(a new virtual machine for the network\). These upgrades and new features are developed by a decentralized network of core developers spanning 8 individual Ethereum software client teams, the non-profit Ethereum Foundation, and additional volunteer developers and researchers from around the globe.
```
Comment (ChazSchmidt) [inline other/ethhub-cftc-response.md]: I think it is useful to mention that the core development spans multiple teams in a decentralized way.

File: other/ethhub-cftc-response.md [hunk 3]
```diff
@@ -10,11 +10,16 @@ DEADLINE: February 9, 2019
 
 ### 1. What was the impetus for developing Ether and the Ethereum Network, especially relative to Bitcoin?
 
-It's first vitally important to distinguish between Ether and Ethereum. Ethereum is an open-source, blockchain-based distributed ledger with decentralized control that supports general computing and economic activities. Anyone is able to build and deploy decentralized products and services that run on top of Ethereum by creating highly flexible programmable transactions, called 'smart contracts'. This smart contract technology has proven to be attractive to developers since it enables them to create programs and business logic that run exactly as coded, trustlessly and with no down time.
+The underlying impetus to develop Ethereum was to utilize aspects of the technology initially developed as part of the Bitcoin blockchain and combine it with the capabilities of a Turing complete supporting environment.
```
Comment (ChazSchmidt) [inline other/ethhub-cftc-response.md]: Turing complete is useful here. Might need to define Turing complete for the laymen though.

File: other/ethhub-cftc-response.md [hunk 4]
```diff
@@ -10,11 +10,16 @@ DEADLINE: February 9, 2019
 
 ### 1. What was the impetus for developing Ether and the Ethereum Network, especially relative to Bitcoin?
 
-It's first vitally important to distinguish between Ether and Ethereum. Ethereum is an open-source, blockchain-based distributed ledger with decentralized control that supports general computing and economic activities. Anyone is able to build and deploy decentralized products and services that run on top of Ethereum by creating highly flexible programmable transactions, called 'smart contracts'. This smart contract technology has proven to be attractive to developers since it enables them to create programs and business logic that run exactly as coded, trustlessly and with no down time.
```
Comment (ChazSchmidt) [inline other/ethhub-cftc-response.md]: It is useful to distinguish between Ether and Ethereum right off the bat for those who do not know the difference. Mention of smart contracts for the laymen is useful as well. Maybe mention Turing completeness enables smart contracts.

File: other/ethhub-cftc-response.md (hunk 3 above)
Comment (ChazSchmidt) [reply other/ethhub-cftc-response.md]: I saw your sources down below. I think it is still use to give an overview of Turing completeness in addition to providing your sources.

File: other/ethhub-cftc-response.md (hunk 4 above)
Comment (cyber-hokie) [reply other/ethhub-cftc-response.md]: It’s not as important to define Ether vs Ethereum as the questions later lend to the impression that they already know this. It is important to include more information than this author wants to update this to. For example, the impetus is not just coopting Bitcoin tech and slapping on a quasi-turing complete programming language and execution environment. Bitcoin Script is not expressive or flexible enough to support many of the use cases Ethereum’s creators envisioned. Additionally, an unwillingness of the Bitcoin core client maintainers to enable opcodes and discuss block size increases made the Bitcoin blockchain a non-starter for those same use cases. This is all relevant and lends to the motivation for developing a separate network vs simply building on Bitcoin a la RSK.

Comment (ChazSchmidt) [reply other/ethhub-cftc-response.md]: >For example, the impetus is not just coopting Bitcoin tech and slapping on a quasi-turing complete programming language and execution environment. Bitcoin Script is not expressive or flexible enough to support many of the use cases Ethereum’s creators envisioned. Additionally, an unwillingness of the Bitcoin core client maintainers to enable opcodes and discuss block size increases made the Bitcoin blockchain a non-starter for those same use cases. This is all relevant and lends to the motivation for developing a separate network vs simply building on Bitcoin a la RSK.

Great point. We need to keep in mind that they want to know the practical reason of why Ethereum came into existence.

Comment (chanan-sack) [reply other/ethhub-cftc-response.md]: thanks for the input, my opinion here is that we need to prioritize changes in this document - incorrect information should be fixed before further explanations for the layman (even though they are important).
this is why i think my pull request should be merged, later anyone and myself will be able to add and contribute to a correct answer and make it better.

**Pull request closed without merging**